iot_config/src/route.rs

```diff
@@ -7,7 +7,7 @@ use helium_proto::Region;
 use serde::{Deserialize, Serialize};
 use serde_json::json;
 use sqlx::{types::Uuid, Row};
-use std::{collections::BTreeMap, sync::Arc};
```

Comment:
should update the PR description now that the arc is dropped

Reply:
Good catch